submit-web/src/components/UserManagement/NewUser/FormOptions.tsx

```diff
@@ -7,37 +7,42 @@ type FormOptionsProps = {
   disabled?: boolean;
 };
 
+export enum Role {
```

Comment:
this seems useful across the app, can we take it to constant file, somewhere shared and common